.yarnrc

```diff
@@ -0,0 +1 @@
+registry "https://registry.yarnpkg.com"
```

Comment:
`yarn config get registry` was not respecting `npmrc` for some reason, so added `.yarnrc`